src/store/store.js

```diff
@@ -0,0 +1,11 @@
+import thunk from 'redux-thunk';
+import { createStore, applyMiddleware, compose } from 'redux';
+import reducer from './rootReducer';
+
```

Comment:
```suggestion
import { composeWithDevTools } from "redux-devtools-extension";
```

Reply:
Duly noted @kcharles52. I will be using `composeWithDevTools`